modules/fedimint-ln-client/src/lib.rs

```diff
@@ -618,16 +618,8 @@ impl LightningClientModule {
             secp,
             client_ctx: args.context(),
             update_gateway_cache_merge: UpdateMerge::default(),
-            gateway_conn: gateway_conn.clone(),
-        };
-
-        // Only initialize the gateway cache if it is empty
```

Comment:
Not doing it here is a significant change API change. I don't have a full understanding of the consequences.

But generally yes, this needs to be fixed, and doing rpc calls here slows the client initialization, and possibility of failing whole client due to a connectivity problem is terrible.

Alternatively, we should spawn a background task or something to keep doing it. Though the downside is that it racy and non-deterministic, so I'm a bit meh on it.